Glossary.md

```diff
@@ -0,0 +1,149 @@
+
+## Consensus Time
+**Consensus Time**: The timestamp assigned to a transaction once it has reached consensus across the network. This timestamp indicates the moment when the network agrees on the order and validity of the transaction, ensuring that all participants have a consistent view of the transaction sequence.
+
+---
+
+## Delegate Call
+**Delegate Call**: Used to call a function in another contract, but with the context of the calling contract. This allows for the execution of functions in one contract as if they were part of another but retains the original caller’s context (msg.sender and msg.value).
+
+---
+
+## EOA
+**EOA (Externally Owned Accounts)**: Accounts that are controlled by private keys and are not associated with smart contracts. These accounts are used to send and receive transactions on the network and interact with smart contracts. EOAs are typically owned by individual users and are used to manage their assets and participate in the network.
+
+---
+
+## Hollow Accounts
+**Hollow Accounts**: An account that has been created with an account number and alias but lacks an account key. This term is specifically used to describe accounts generated through the Auto Account Creation feature, which allows applications to create user accounts instantly, even without an internet connection, by assigning an account alias.
+
+---
+
+## Leaky Tests
+**Leaky Tests**: Are unit or other tests that should not be run in parallel because they can “leak” some of their state or data out into the global state.
+
+---
+
+## Precompiles
+**Precompiles**: Contracts that are compiled and deployed on the blockchain before they are executed. They are used to optimize the execution of certain operations that are computationally expensive or require a large amount of gas to execute. Precompiled contracts are typically used to perform cryptographic operations, such as hashing, encryption, and decryption, that are required for the operation of the blockchain.
+
+---
+
+## Security Model V2
+**Security Model v2**: Hedera introduced the HSCS Security Model v2 to enhance the security of its network building with improved features to better safeguard against potential vulnerabilities and attacks.  By incorporating enhanced cryptographic techniques, robust consensus mechanisms, decentralized governance, and continuous monitoring.
+Key features include - Restrictions on Smart contracts Storage; Prohibition on Delegate Calls to System Smart Contracts; Limited Access to EOAs' Storage; Token Allowance Requirement for Balance Changes
+
+## Sidecars
+**Sidecars**: In Services, a sidecar refers to additional records that are created alongside the main transaction records. These sidecar records provide more detailed information about the transactions, making it easier to debug and understand the state changes that occur as a result of the transactions.
+
+## Synthetic Transactions
```

Comment:
An example or two might be useful.  E.g., it's a couple of synthetic transactions that causes the network to freeze at a specific time so it can be upgraded.

Also, confusing to me, I see uses of "synthetic transaction" in the docs where I would have expected to see "child transaction".  I might have to learn more about what's intended by this.